corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -790,7 +829,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
 
         self.hasAnyNestedQuestions = function () {
             return _.any(self.children(), function (d) {
-                if (d.type() === constants.QUESTION_TYPE || d.type() === constants.REPEAT_TYPE) {
+                if (d.type() === constants.QUESTION_TYPE || d.type() === constants.REPEAT_TYPE || d.type() === "add-group") {
```

Comment:
`add-group` probably ought to be a constant, too

Reply:
fair. did not need that clause after all though. Fixed by deletion ;)